packages/compass-crud/src/components/bulk-delete-modal.tsx

```diff
@@ -16,6 +16,7 @@ import type { BSONObject } from '../stores/crud-store';
 import { toJSString } from 'mongodb-query-parser';
 import { ReadonlyFilter } from './readonly-filter';
 import ReadonlyDocument from './readonly-document';
+import type { Document } from 'bson';
```

Comment:
wait .. how does this change work?

Reply:
Ah, sorry, should've left a comment here, it is indeed very obscure otherwise. This is a drive-by fix, part of refactoring document list to function component. `BulkDelete` modal is using `Document` type for one of the component props:

https://github.com/mongodb-js/compass/blob/4cdba20af897b9bb49c16d7afac2d6ed65e861ff/packages/compass-crud/src/components/bulk-delete-modal.tsx#L66

But there was no type import, so it was referring to the DOM namespace type of `window.document`. And then the component was using `as any` assertion to work around a valid type issue 🙈 

https://github.com/mongodb-js/compass/blob/4cdba20af897b9bb49c16d7afac2d6ed65e861ff/packages/compass-crud/src/components/document-list.tsx#L319-L321

So adding this import here just fixes the type issue. I'm pretty sure that was the intended `Document` type here

Reply:
ahh ok, thanks!